### PR TITLE
Translate part of knowledge/javascript-conventions into portuguese (pt-br)

### PR DESCRIPTION
**Pull request by ftonato**
Hello,

I have finished the `knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md` file, can you validate and approve this one?

/cc @nodejs/nodejs-pt
/cco @glaucia86 @khaosdoctor

### CODE REVIEW

File: locale/pt-br/knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+false // obviamente
+0     // o único number considerado "falsy"
+""    // uma string vazia
+null
+undefined
+NaN
+```
+    
+<!-- Note that all objects and arrays are truthy, even empty ones. -->
+Note que todas os objetos e arrays são truthy, ainda que sejam vazios.
```
Comment (khaosdoctor) [inline locale/pt-br/knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md]:
```suggestion
Note que todos os objetos e arrays são truthy, ainda que sejam vazios.
```

File: locale/pt-br/knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+<!-- Truthiness and Falsiness also come into play with logical
+operators. When using logical AND/OR, the values will be converted
+based on truthiness or falseness and then the expression will resolve
+to the last truthy value. Short circuit rules apply. Here's an
+extended example. -->
+Truthiness e Falsiness também entram em jogo com operadores lógicos. Quando usamos operadores lógicos AND/OR, os valores serão convertidos com base em truthiness ou falseness e, em seguida, a expressão será resolvida para o último valor do truthy. Regras de curto-circuito se aplicam. Aqui está um breve exemplo.
```
Comment (khaosdoctor) [inline locale/pt-br/knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md]:
```suggestion
Truthiness e Falsiness também entram em jogo com operadores lógicos. Quando usamos operadores lógicos AND/OR, os valores serão convertidos com base em truthiness ou falsiness e, em seguida, a expressão será resolvida para o último valor truthy. Regras de curto-circuito se aplicam. Aqui está um breve exemplo.
```

File: locale/pt-br/knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+title: 'O que são valores "truthy" e "falsy"?'
+difficulty: 4
+layout: knowledge-post.hbs
+---
+
+
+<!-- JavaScript is weakly typed language. That means different types can be
+used in operations and the language will try to convert the types
+until the operation makes sense. -->
+JavaScript é uma linguagem fracamente tipada. Isso significa que diferentes tipos podem ser usados em operações e a linguagem tentará converter os tipos até que a operação faça sentido.
```
Comment (khaosdoctor) [inline locale/pt-br/knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md]:
```suggestion
JavaScript é uma linguagem fracamente tipada. Isso significa que diferentes tipos podem ser usados em operações e a linguagem tentará converter estes tipos até que a operação faça sentido.
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
Did some comments

**Comment by ftonato** [comment]
> Did some comments

Hello, I have fixed right now, tks :)

**Review by khaosdoctor — APPROVED** [review]
Awesome! LGTM 👍 

cc: @Trott @MaledongGit

**Comment by ghost** [comment]
Please also see #2278. We don't need to do translations, if you have other feedbacks, please leave them here :)

**Pull request closed without merging**